.flake8

```diff
@@ -1,19 +1,59 @@
 [flake8]
-# E203 skipped because it annoys black.
-# no pyproject.toml support https://gitlab.com/pycqa/flake8/-/issues/428
-show-source = true
-extend-ignore =
-  E203,
-  F401,
-max-line-length = 100
+
 builtins = _
+
+# Print the total number of errors:
+count = true
+
+# Don't even try to analyze these:
 extend-exclude =
-  # skip tool cache dirs
+  # No need to traverse egg info dir
+  *.egg-info,
+  # tool cache dirs
   *_cache
-  # skip project env vars
+  # project env vars
   .env,
-  # skip adjacent venv
+  # GitHub configs
+  .github,
+  # Cache files of MyPy
+  .mypy_cache,
+  # Cache files of pytest
+  .pytest_cache,
+  # Temp dir of pytest-testmon
+  .tmontmp,
+  # Occasional virtualenv dir
+  .venv
+  # VS Code
+  .vscode,
+  # Temporary build dir
+  build,
+  # This contains sdists and wheels of ansible-navigator that we don't want to check
+  dist,
+  # Metadata of `pip wheel` cmd is autogenerated
+  pip-wheel-metadata,
+  # adjacent venv
   venv
+
+# IMPORTANT: avoid using ignore option, always use extend-ignore instead
+# Completely and unconditionally ignore the following errors:
+extend-ignore =
+  E203,  # skipped because it annoys black
+  F401,  # skipped for unknown reason
+  WPS300,  # "Found local folder import" -- nothing bad about this
+  WPS306,  # "Found class without a base class: *" -- irrelevant under Python 3
+  WPS422,  # "Found future import: *" -- we need these for multipython
+# https://wemake-python-stylegui.de/en/latest/pages/usage/formatter.html
+#format = wemake
+
+# Let's not overcomplicate the code:
+#max-complexity = 10
+
+# Accessibility/large fonts and PEP8 friendly:
+#max-line-length = 79
+# Accessibility/large fonts and PEP8 unfriendly:
+max-line-length = 100
+
+# Allow certain violations in certain files:
```

Comment:
Can we add a note in the global section that:

1) exceptions should be fixed
2) if not, exception is in the file, inline
3) if not possible for some reason, exception can go here, but a majority of these are temporary
4) ACD of global needs team review (not today but in the future)

Reply:
Such comments should be present below, once the entries are separated into what needs to be fixed vs what needs to remain ignored. But this is not the part of this PR because its sole purpose is to sync the config structure.
If we were to improve the annotations above the settings, this would be separate and synched back into other projects with a similar baseline configuration. Also, what do you mean by global? At the top of the config file?

Reply:
yeah, sorry, someplace obvious that would indicate that changes to the file are generally not necessary for exceptions as explained above.  

If not in this PR, yeah you guessed it, a followup issue & PR?

Reply:
https://github.com/ansible/ansible-navigator/issues/722